mirage/factories/registration.ts

```diff
@@ -154,6 +154,19 @@ export default NodeFactory.extend<MirageRegistration & RegistrationTraits>({
         return faker.internet.url();
     },
 
+    providerSpecificMetadata() {
```

Comment:
I think this should be empty by default and have a trait `hasProviderMetadata` or something like that to populate it. Also, if the provider doesn't have its own metadata, is `providerSpecificMetadata` `null` or `[]`?

Reply:
If we do that, which makes sense, we should have at least one example of filled out metadata in the mirage defaults.

Reply:
Jon says empty response is `[]`